### ISSUE TITLE
Would you like to see support for extra gravities?

### ISSUE THREAD

**Issue by jaz303** [jaz303/tipsy#12]
i.e. 8 compass points instead of 4

I've been thinking about this for ages now, pretty much since I released the project. If enough people vote this up I'll look at adding it in.

-J

**Comment by JakeWharton** [comment]
Can I also propose that you include NNE, NNW, SSE, and SSW as well?

```
 _________
|_________|
  V
 <a>
 _________
|_________|
        V
       <a>

 <a>
 _^_______
|_________|

       <a>
 _______^_
|_________|
```

I figure since tooltips are mostly fat and not tall ENE, ESE, WSW, and WNW can be omitted.

**Comment by jaz303** [comment]
Please excuse my inaccurate description - this is exactly what I was imagining.

**Comment by rennercr** [comment]
Yes please.  Desperately need this feature!

**Comment by jaz303** [comment]
support for 8 compass points (closed by 12db0b465febb1ebadb0e9f528737384cbb2350b)
